testdata/google/events/cloud/audit/v1/LogEntryData-issue43.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
For the file name, I think issue 43 will be long gone from most users.

We should just have something like `LogEntryData-storageBbucketsUpdate` to provide a searchable name.

Reply:
Happy to rename all of these, definitely.